docs/api-ref.md

```diff
@@ -136,18 +133,35 @@ REST API: [Sign Out](https://help.tableau.com/current/api/rest_api/en-us/REST/re
 **Example**
 
 ```py
-
 server.auth.sign_out()
+```
+
+<br>
+<br> 
 
+#### auth.switch_site
 
+```py
+auth.switch_site(site_id)
 ```
 
+Switch to a different site on the current Tableau Server.
 
+Switching avoids the need for reauthenticating to the same server. (Note: This method is not available in Tableau Online.)
 
+REST API: [Switch Site](https://help.tableau.com/current/api/rest_api/en-us/REST/rest_api_ref_authentication.htm#switch_site)
 
-**See Also**  
-[Sign in and Out](sign-in-out)  
-[Server](#server)
+**Parameters**
+
+`site_id` | This corresponds to the `contentUrl` attribute in the Tableau REST API. The `site_id` is the portion of the URL that follows the `/site/` in the URL. For example, "MarketingTeam" is the `site_id` in the following URL *MyServer*/#/site/**MarketingTeam**/projects.
```

Comment:
This method takes in a site_item, which contains the contentUrl field.

Reply:
Whoops, totally got that wrong. Fixing now with a better example too.